scripts/apps/authoring/authoring/services/RenditionsService.js

```diff
@@ -74,17 +74,28 @@ export function RenditionsService(metadata, $q, api, superdesk, _) {
                 withRatio = self.renditions;
             }
 
+            const cropOptions = { // default options
+                ...{
```

Comment:
Why do you need to use the spread operator here instead of listing the properties?

Reply:
@tomaskikutis you're right, my bad